org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -213,9 +213,6 @@ public void triggerInitialization(Collection<IPath> roots) {
 		JavaLanguageServerPlugin.logInfo("ProjectRegistryRefreshJob finished " + (System.currentTimeMillis() - start) + "ms");
 		// load gradle plugin https://github.com/redhat-developer/vscode-java/issues/2088
 		startBundle(CorePlugin.PLUGIN_ID);
-		start = System.currentTimeMillis();
```

Comment:
I found the wait part comes from https://github.com/eclipse/eclipse.jdt.ls/pull/1874, since the LoadingGradleVersionJob will be no longer conflicting via https://github.com/eclipse/buildship/commit/2d105739966ef4d07dc12b368d22553f8efcffc4, @snjeza can we move this part? Or am I missing something here? It would be great if there is a way to test this.

Reply:
@CsCherrYY I think we can. I can't reproduce the issue described at https://github.com/redhat-developer/vscode-java/issues/2088#issue-981412537 with your patch.
I'm not sure how to create a test for it. The issue happened randomly when Java LS tarted.